docs/source/performance.rst

```diff
@@ -131,4 +131,65 @@ To use Optimizer Sharded Training, refer to :ref:`model-parallelism`.
 
 Sharded DDP can work across all DDP variants by adding the additional ``--plugins ddp_sharded`` flag.
 
-Refer to the :ref:`distributed computing guide for more details <multi_gpu>`.
\ No newline at end of file
+Refer to the :ref:`distributed computing guide for more details <multi_gpu>`.
+
+
+Sequential Model Parallelism with Checkpointing to reduce peak memory
+---------------------------------------------------------------------
+Pipe Pipeline is a lightning integration of Pipeline Parallelism provided by Fairscale.
+Pipe combines pipeline parallelism with checkpointing to reduce peak memory required to train while minimizing device under-utilization.
+
+Find more explanation at https://arxiv.org/abs/1811.06965
```

Comment:
As done for sharded, I'd add a cross reference to the docs rather than duplicating everything here.

Keep the memory benchmark, but cross reference everything else